Qa/kommander 1 4 kubecost (#3561)
mesosphere/dcos-docs-site: pages/dkp/kommander/1.4/centralized-monitoring/index.md

### Federating Prometheus Alerting Rules

You can define additional [Prometheus alerting rules][alerting_rules] on the Kommander cluster and federate them to all of the managed clusters by following these instructions.

> In order to use these instructions, you will need to [install the kubefedctl CLI][kubefedctl].

Reviewer: ```suggestion
To use these instructions you must [install the kubefedctl CLI][kubefedctl].
```
